tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/SeleniumTests.cs

```diff
@@ -0,0 +1,163 @@
+// <copyright file="SeleniumTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.Ci.CiEnvironment;
+using Datadog.Trace.Ci.Tags;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.TestHelpers.Ci;
+using Datadog.Trace.Util;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using FluentAssertions;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.CI;
+
+[UsesVerify]
+public class SeleniumTests : TestingFrameworkEvpTest
+{
+    public SeleniumTests(ITestOutputHelper output)
+        : base("Selenium", output)
+    {
+        SetServiceName("xunit-selenium-tests");
+        SetServiceVersion("1.0.0");
+    }
+
+    [SkippableTheory]
+    [MemberData(nameof(PackageVersions.Selenium), MemberType = typeof(PackageVersions))]
+    [Trait("RunOnWindows", "True")]
+    [Trait("Category", "EndToEnd")]
+    [Trait("Category", "TestIntegrations")]
+    public async Task Injection(string packageVersion)
+    {
+        SkipOn.Platform(SkipOn.PlatformValue.Linux);
```

Comment:
Interesting? 🤔

Reply:
Selenium suck and I was unable to test it in Linux, I'm getting File not found or Permission denied exceptions when creating the chrome driver.

Reply:
I suspect it's related to the fact we're running in Docker?